packages/home-service/README.md

```diff
@@ -0,0 +1,25 @@
+Home Microservice
+=================================================
+
+One platform's server-side Home GraphQL microservice.
```

Comment:
Please add more information regarding your home spa

Reply:
```
The Home SPA needs a backend API Service to fetch the microservice details and the Apps in spotlight list + videos. This microservice would solve that problem
by storing all such information. 
```
Added to readme